frontend/src/views/WheresMyGene/components/HeatMap/components/YAxisChart/index.tsx

```diff
@@ -216,9 +214,12 @@ export default memo(function YAxisChart({
       {yAxisInfoCoords &&
         isMarkerGenes &&
         !FMG_EXCLUDE_TISSUES.includes(tissue) &&
+        cellTypeMetadata.length > 0 &&
```

Comment:
fixes #3849?

Reply:
This quick fix stops the runtime error, but  do we know why `cellTypeMetadata` can be an empty array or undefined here? In my mind every cell type should always have serialized metadata, so `cellTypeMetadata` should always be available 🤔

Reply:
Just for posterity I followed the thread earlier when I encountered the bug and saw that `cellTypeMetadata` is empty when `cellTypes` is empty

Reply:
Thanks for digging into this! 

Interesting, the surrounding code was introduced when the FMG info button was added, how come we're only seeing the bug now? And does the runtime error only happen for certain datasets or all of them? 

IIRC the info buttons were rendering just fine before the holidays, so not sure what code change caused the runtime error now 🤔

Reply:
CC @atarashansky in case you have more info here! Thanks!

Reply:
fix is void from upstream changes on main